R/derive_param_computed.R

```diff
@@ -188,8 +203,23 @@
 #'   )
 #' )
 #'
-#' # Example 1b: Using option `keep_nas = TRUE` to derive MAP in the case where some/all
-#' # values of a variable used in the computation are missing
+#' @caption Example 1b - Adding a parameter computed from a formula with `keep_nas = TRUE`
+#'
+#' @info Using option `keep_nas = TRUE` to derive MAP in the case
+#'     where some/all values of a variable used in the computation are missing
+#'
+#' - Note that observations will be added here even if some of the values contributing
+#'   to the computed values are `NA`.
+#' - This therefore results in four additional rows: `USUBJID` `01-701-1015`
```

Comment:
Again, don't need to explicitly mention the rows - just say there is one more row for the patient who has an NA valye for SYSBP (or similar)